README.md

```diff
@@ -19,6 +76,29 @@ The source code can be validated (golint, gofmt, unit testing) running:
 make validate test
 ```
 
+## Build Subctl
+
+To build subctl locally
+```bash
+make build-subctl
```

Comment:
This should be

    make bin/subctl

`build-subctl` has issues currently.

Reply:
Done.

Reply:
I found in #166  that it is also required to run: `go mod vendor` before `make bin/subctl`, otherwise build may fail on: `build command-line-arguments: cannot load github.com/kubernetes-sigs/kubefed/pkg/version: open /go/src/github.com/submariner-io/submariner-operator/vendor/github.com/kubernetes-sigs/kubefed/pkg/version: no such file or directory`

Reply:
Done.